### PR TITLE
Add elementary system Flatpak repo

### PR DESCRIPTION
**Pull request by danrabbit**
- [x] Needs GPG key

### CODE REVIEW

**Comment by danrabbit** [comment]
Assigned Blake for the actual location of the repo files

**Comment by davidmhewitt** [comment]
I don't know whether there was a decision whether these should be system or user remotes. But, adding these here will make them system remotes. We'd need a different approach if we wanted one/both of these to be a user remote.

**Comment by btkostner** [comment]
So, from what I know and understand, we have the elementary repo at `https://flatpak.elementary.io/elementary` which will be system, and then `https://flatpak.elementary.io/appcenter` which will be a user remote

**Comment by davidmhewitt** [comment]
I think the solution to adding a user remote will be to add it to a working system and figure out what config file gets created in the home directory when that happens. It's a keyfile somewhere, I remember finding it once.

Then we can just add that to `/etc/skel` in the image.

File: etc/config/hooks/live/001-install-flatpaks.chroot [hunk 1]
```diff
@@ -16,5 +16,4 @@ flatpak remote-add --if-not-exists --filter=/etc/flatpak/epiphany.filter flathub
 
 flatpak install -y flathub org.gnome.Epiphany
 
-flatpak remote-add --if-not-exists appcenter https://flatpak.elementary.io/appcenter/
-flatpak remote-add --if-not-exists elementary https://flatpak.elementary.io/elementary/
+flatpak remote-add --if-not-exists elementary https://flatpak.elementary.io/elementary/elementary.flatpakrepo
```
Comment (meisenzahl) [inline etc/config/hooks/live/001-install-flatpaks.chroot:19]:
```suggestion
flatpak remote-add --if-not-exists elementary https://flatpak.elementary.io/elementary.flatpakrepo
```

Comment (meisenzahl) [reply etc/config/hooks/live/001-install-flatpaks.chroot:19]: Shouldn't it be https://flatpak.elementary.io/elementary.flatpakrepo? https://flatpak.elementary.io/elementary/elementary.flatpakrepo returns a 404.

**Review by meisenzahl — CHANGES_REQUESTED** [review]

**Review by btkostner — APPROVED** [review]
Confirmed working on my local computer now. We should be good to merge.

**Comment by btkostner** [comment]
Can I get one other person to confirm they are able to add the repo and install platform or calculator. I was getting a runtime error with calculator, but install worked fine.

**Comment by danrabbit** [comment]
@btkostner Yup! I was able to add this repo and install calculator. Congrats! You are the man!

**Pull request merged**